### DISCUSSION TITLE
SyntaxError: Unexpected token 'export'. import { dia, ui, shapes } from '@clientio/rappid';

### DISCUSSION THREAD

**Discussion by LesiaMat95** [clientIO/joint#2045] [Q&A] [answered]
I deployed chatBot locally. I wrote the first jest test and faced a problem that exists only at the time of testing the project. 
the Chatbot code is derived from JointJS+. This code hasn't even changed yet. The project is launched, everything works personally, but with unit tests trouble. I tried to mock svg since you have already discussed this topic but it did not help me. 
![image](https://user-images.githubusercontent.com/125379772/218773081-1652e9f0-e756-4839-9b44-f4096c0a776f.png)

**Comment by jamesgeorgewilliams** [comment, marked as answer]
This error is usually related to `transformIgnorePatterns` in your `Jest` config file.

We have fixed the same issue with that option in our JointJS and React Application. It was necessary for us to add the option via an npm script as we were using create-react-app.
https://github.com/clientIO/joint-plus-tutorial-react/pull/3
https://github.com/clientIO/joint-plus-tutorial-react/blob/main/package.json

You can find some similar information at the following links:
https://stackoverflow.com/questions/52035066/how-to-write-jest-transformignorepatterns
https://github.com/nrwl/nx/issues/812

**Comment by LesiaMat95** [comment]
Thank you! This solved my problem!

**Reply by jamesgeorgewilliams** [reply]
Great! Thanks for letting us know.
